lib/fastlane/actions/adb.rb

```diff
@@ -0,0 +1,65 @@
+module Fastlane
+  module Actions
+    module SharedValues
+    end
+
+    class AdbAction < Action
+      def self.run(params)
+        adb = Helper::AdbHelper.new(adb_path: params[:adb_path])
+        result = adb.trigger(flags: params[:flags], serial: params[:serial])
+        return result
+      end
+
+      #####################################################
+      # @!group Documentation
+      #####################################################
+
+      def self.description
+        "Run ADB Actions"
+      end
+
+      def self.details
+        [
+          "see adb --help for more details"
+        ].join("\n")
+      end
+
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(key: :serial,
+                                       env_name: "FL_ANDROID_SERIAL",
+                                       description: "Android serial, wich device should be used for this command",
+                                       is_string: true,
+                                       default_value: ""),
+          FastlaneCore::ConfigItem.new(key: :flags,
```

Comment:
Can we call this `command` instead of `flags`? The values passed to `adb` are [documented](http://developer.android.com/tools/help/adb.html) as commands.